common/extensions/api/_brave_wallet_api_features.json

```diff
@@ -24,42 +24,6 @@
     ],
     "default_parent": true
   }],
-  "braveWallet.getWalletSeed": {
-    "channel": "stable",
-    "dependencies": [],
-    "contexts": ["privileged_extension"],
-    "component_extensions_auto_granted": false,
-    "allowlist": [
-      "21070F3D60711361C1210B870439BE49B5D995F4"
```

Comment:
Should we also clear all references to `21070F3D60711361C1210B870439BE49B5D995F4` in this file?

Reply:
I did that and from all other files too. Please review this commit for that:
4a12d50a1f4aba02fbd49bb4dfb0ac6957c066e8